lib/oli/automation_setup.ex

```diff
@@ -0,0 +1,354 @@
+defmodule Oli.AutomationSetup do
+  @moduledoc """
+  Sets up data in a torus instance that is meant be used with automated e2e tests to
+  enable testing of content in remote torus instances.
+  """
+  alias Lti_1p3.Tool.ContextRoles
+
+  alias Oli.Repo
+  alias Oli.Authoring.Course.Project
+  alias Oli.Authoring.Course.ProjectResource
+
+  import Ecto.Query, warn: false
+  require Logger
+
+  def setup_data(
```

Comment:
I wonder if we should enclose this function in a transaction, since any of the steps could fail and the user wouldn't know which records were created and which ones weren't

Reply:
Got that, and also did project teardown in a transaction.